src/note_automatic_jete.lua

```diff
@@ -63,11 +63,11 @@ local articulation = require("library.articulation")
 local configuration = require("library.configuration")
 local library = require("library.general_library")
 
-local max_layers = 4            -- this should be in the PDK, but for some reason isn't
+local max_layers = 4 -- this should be in the PDK, but for some reason isn't
```

Comment:
The comment is missing the word "Framework". It should say:

```
this should be in the PDK Framework, but for some reason isn't
```

I'll make a note to add it whenever I do the next version.